src/dev/register_git_hook/register_git_hook.js

```diff
@@ -124,11 +124,12 @@ export async function registerPrecommitGitHook(log) {
   );
 
   try {
+    const homedir = process.env.HOME ? process.env.HOME.replace(/'/g, '\'') : '~';
```

Comment:
Can we do something like 
`const homedir = process.env.HOME ? process.env.HOME.replace(/'/g, '\'') : process.env.USERPROFILE.replace(/'/g, '\'');` instead?

Reply:
Thanks for the suggestion. I have revised the patch and rebased it to the master.